tests/ert3/data/test_transmitters.py

```diff
@@ -0,0 +1,141 @@
+import contextlib
```

Comment:
I'd like to see tests displaying that both `load` and `dump` is unexpected for not transmitted transmitters...